### DISCUSSION TITLE
Why does it crash when I add motor3 model simulation on the basis of IEEE14_full?

### DISCUSSION THREAD

**Discussion by wangxiaowu-sudo** [CURENT/andes#582] [Q&A]
I added ZIP model, fault model, and motor3 model on the basis of IEEE_full. The first two models can simulate normally when added, but after adding motor3, they can run normally
import andes
from andes.utils.paths import get_case
andes.config_logger(stream_level=20)
ss = andes.run(get_case('/usr/local/lib/python3.10/dist-packages/andes/cases/ieee14/ieee14_full.xlsx'),default_config=True)  # one can remove `default_config=True` to use custom config file
Always collapsing
[ieee14_full (1).xlsx](https://github.com/user-attachments/files/17972190/ieee14_full.1.xlsx)

**Comment by jinningwang** [comment]
The power flow did not converge, I would suggest tune the parameters to converge power flow first.
